bookkeeper-server/src/main/java/org/apache/bookkeeper/replication/Auditor.java

```diff
@@ -587,7 +270,7 @@ synchronized Future<?> submitAuditTask() {
 
                 knownBookies.removeAll(bookiesToBeAudited);
                 if (lostBookieRecoveryDelay == 0) {
-                    startAudit(false);
+                    newAuditorBookieCheckTask().startAudit(false);
```

Comment:
Are there any ways that do not create the `checkTask` instance every time?

Reply:
@hangc0276 May be we can use netty Recycler. But I don't think a large number of AuditorTasks will be created in a short period of time, because the inspection tasks are executed periodically, most of them are once a day, and these instances can be recycled by the GC as soon as possible.

If you feel strongly about this, I can add.

Reply:
Maybe we can reuse the one AuditorTask, I'll try it.